fe1-web/src/App.tsx

```diff
@@ -29,6 +30,42 @@ configureNetwork(messageRegistry, keyPairRegistry);
 // start persisting the redux state after all reducers have been registered
 persist.persist();
 
+const BuildInfo = () => {
+  const styles = StyleSheet.create({
+    container: {
+      position: 'absolute',
+      bottom: '4px',
+      left: '3px',
+      zIndex: 100,
+      color: '#757575',
+      fontSize: '8px',
+      fontFamily: 'monospace',
+      textTransform: 'uppercase',
+    },
+    link: {
+      textDecorationLine: 'none',
+      color: '#757575',
+    },
+  });
+
+  return (
+    <div style={styles.container}>
+      <a
+        style={styles.link}
+        href={`https://github.com/dedis/popstellar/releases/tag/${Constants?.expoConfig?.extra?.appVersion}`}
+        target="_blank">
+        {Constants?.expoConfig?.extra?.appVersion}
+      </a>
+      {' | '}
+      <a style={styles.link} href={Constants?.expoConfig?.extra?.buildURL} target="_blank">
+        {Constants?.expoConfig?.extra?.shortSHA}
+      </a>
+      {' | '}
+      {Constants?.expoConfig?.extra?.buildDate}
+    </div>
```

Comment:
If we want to maintain compatibility with native builds we cannot use standard HTML elements such as `div` or `a` or even plain unwrapped text.

react-native's `View` more or less corresponds to the HTML `div` and all texts have to be wrapped in `<Text>xyz</Text>` elements.

> The most fundamental component for building a UI, View is a container that supports layout with [flexbox](https://reactnative.dev/docs/flexbox), [style](https://reactnative.dev/docs/style), [some touch handling](https://reactnative.dev/docs/handling-touches), and [accessibility](https://reactnative.dev/docs/accessibility) controls. View maps directly to the native view equivalent on whatever platform React Native is running on, whether that is a UIView, \<div>, android.view, etc.

(https://reactnative.dev/docs/view)

For displaying a link it becomes even more cumbersome:

```jsx
<Text onPress={() => Linking.openURL('http://google.com')}>Google</Text>
```

(From https://stackoverflow.com/a/30540502/2897827)